### ISSUE TITLE
"Exception reported: no such column: slot" & "Invalid query Handle 0 (error: 4)"

### ISSUE THREAD

**Issue by Missiongames** [FlaminSarge/tf2idb#9]
Getting this error after attempting to load tf2idb.smx on my dedicated server.
Sourcemod version used for compiling: 1.8.0-git5969-windows
Python version used for running script: 2.7.10

```
L 01/30/2017 - 18:55:10: [SM] Exception reported: no such column: slot
L 01/30/2017 - 18:55:10: [SM] Blaming: tf2idb.smx
L 01/30/2017 - 18:55:10: [SM] Call stack trace:
L 01/30/2017 - 18:55:10: [SM]   [0] SetFailState
L 01/30/2017 - 18:55:10: [SM]   [1] Line 114, sourcemod\scripting\tf2idb.sp::OnPluginStart
L 01/30/2017 - 18:55:10: [SM] Unable to load plugin "tf2idb.smx": Error detected in plugin startup (see error logs)
L 01/30/2017 - 18:55:10: [SM] Unable to load plugin "tf2idb_tf2ii_compat.smx": Could not find required plugin "tf2idb"
L 01/30/2017 - 18:55:10: [SM] Unable to load plugin "unusual.smx": Could not find required plugin "tf2idb"
```

Also tried Python version 3.5.0 (64-bit) to run the python script, with same Sourcemod version as previously.
This error showed up instead:

```
L 01/30/2017 - 19:42:13: [SM] Exception reported: Invalid query Handle 0 (error: 4)
L 01/30/2017 - 19:42:13: [SM] Blaming: tf2idb.smx
L 01/30/2017 - 19:42:13: [SM] Call stack trace:
L 01/30/2017 - 19:42:13: [SM]   [0] SQL_FetchRow
L 01/30/2017 - 19:42:13: [SM]   [1] Line 209, sourcemod\scripting\tf2idb.sp::PrepareCache
L 01/30/2017 - 19:42:13: [SM]   [2] Line 148, sourcemod\scripting\tf2idb.sp::OnPluginStart
```

Hosting SRCDS on OS: Windows 10 Home
Link to tf2idb.sq3 file (compiled with Python 2.7.10): https://dl.dropboxusercontent.com/u/86694599/tf2idb_sq3.zip

Link to tf2idb.sq3 file (compiled with Python 3.5.0 (64-bit): https://dl.dropboxusercontent.com/u/86694599/tf2idb_sq3_2.zip

**Comment by fakuivan** [comment]
You should use python3 to build the database. The problem of the handle has to do with a typo on a column name ``SELECT name,val FROM tf2idb_qualities``, here ``val`` should be ``value`` as seen on [this pull request](https://github.com/FlaminSarge/tf2idb/pull/8/commits/1816f585c1f3c20488c4b9877d3a9d310c9ab726). Make sure to apply [this change](https://github.com/FlaminSarge/tf2idb/pull/8/commits/f63f4884e01d2ab4b6b5f0ba8763cfa929ddc72d) too. Thanks for the feedback ❤️

**Comment by Missiongames** [comment]
Thanks a lot for the response, I followed your suggestions and it's working fine now.

**Comment by FlaminSarge** [comment]
Fixed by #10
